service/options_test.go

```diff
@@ -43,7 +43,7 @@ func TestAddModules_Errors(t *testing.T) {
 }
 
 func TestWithLogger_OK(t *testing.T) {
-	logger := ulog.Logger()
+	logger := ulog.Builder().Build()
```

Comment:
We could keep the ulog.Logger(), but its good to follow the Builder().Build() suite in the long run. Providing ulog.Build() function sounds like breaking the established builder pattern.